### PR TITLE
Initial test for TKG

### PR DESCRIPTION
**Pull request by renfeiw**
- test framework
- test base, hierarchy and sliceAndDice

Signed-off-by: renfeiw <renfeiw@ca.ibm.com>

### CODE REVIEW

File: scripts/testTKG/testRunner.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+import argparse
+import subprocess
+import re
+import textwrap
+
+CD_TKG = "cd ../../"
+MAKE_CLEAN = "make clean"
+MAKE_COMPILE = "make compile"
+DYNAMIC_COMPILE = "export DYAMIC_COMPILE=true"
```
Comment (smlambert) [inline scripts/testTKG/testRunner.py:23]:
```suggestion
DYNAMIC_COMPILE = "export DYNAMIC_COMPILE=true"
```

**Review by smlambert — COMMENTED** [review]
Request to fix DYNAMIC_COMPILE name, otherwise looks good!

File: scripts/testTKG/testRunner.py (hunk 1 above)
Comment (renfeiw) [reply scripts/testTKG/testRunner.py]: Thank you! Updated.

**Review by llxia — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]
thanks @renfeiw !

**Pull request merged**